src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PanelMemberExclusions.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.sscs.model.client.JudicialUserBase;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude
+public class PanelMemberExclusions {
+    private YesNo arePanelMembersExcluded;
```

Comment:
Extremely minor. Could this be reordered to have the YesNo values next to one another? Fine if not changed.

Reply:
done